content/en/tracing/setup_overview/compatibility_requirements/dotnet-framework.md

```diff
@@ -43,9 +43,11 @@ The .NET Tracer can instrument the following libraries automatically:
 | ADO.NET                         | built-in                       | `AdoNet`             |
 | HttpClient / HttpMessageHandler | built-in                       | `HttpMessageHandler` |
 | WebClient / WebRequest          | built-in                       | `WebRequest`         |
+| Microsoft Message Queuing (MSMQ)| `System.Messaging` 4.0+        | `Msmq`               |
```

Comment:
I think `System.Messaging.dll` is built into .NET Framework (see the few integrations above with `built-in`).

Reply:
As these require customers to enable calltarget (which isn't documented) we will document these in a future release when CallTarget is enabled by default